code/src/components/NewThought.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+
+export const NewThought = ({ onChangeThought }) => {
```

Comment:
There is no onChangeThought function defined anywhere. It seems you wanted to pass it in here as a prop, but it does not exist in App.js either, so even though you call this function later on, nothing happens because the function doesn't exist.
Maybe try writing the function in App and passing it as a prop, or writing it directly in this component 💪